src/utils/isMobile.js

```diff
@@ -0,0 +1,9 @@
+export const isMobile = () => {
+  let isMobile = false
+  if (typeof window !== "undefined") {
+    isMobile = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
+      window.navigator.userAgent
+    )
+  }
```

Comment:
He @ShubhankarKG, thanks for the submission! Looks good, was just wondering if you should consider adding a check for small screen sizes as well in case this missed anything. 

Maybe something like this:
```suggestion
  if (![typeof window, typeof document].includes("undefined")) {
    isMobile = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
      window.navigator.userAgent
    ) | document.documentElement.clientWidth < 1024
  }
```
@pettinarip @corwintines Any thoughts here? Otherwise this looks great =), and would be willing to merge as is.

Reply:
IMO I'd use a well-known lib like https://github.com/faisalman/ua-parser-js for this (e.g. is what `react-detect-device` is using behind the scenes). Why? mainly because we don't need to worry about the maintainability of this function that can grow in complexity over time.

If we decide to go that way, this can be as simple as:
```ts
function isMoblie() {
	return ua.device.type === 'mobile'
}
```

If we keep our custom function, I would argue against checking the size of the screen here. What we really want to check here is if the device is a mobile device regardless of the size of the screen in order to turn on/off some functionality that those devices do/do not support. On the other hand, media queries should be there to control screen sizes and layout design.